<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SdBueBusEditWidget</class>
 <widget class="QWidget" name="C_SdBueBusEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>550</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="pc_VerticalLayout">
     <item>
      <widget class="stw_opensyde_gui_elements::C_OgeTawPageNavi" name="pc_TabWidgetPageNavi">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pc_TabProperties">
        <attribute name="title">
         <string>Properties</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>30</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="pc_VerticalLayout_2">
           <item>
            <widget class="stw_opensyde_gui::C_SdBueBusEditPropertiesWidget" name="pc_BusPropertiesWidget" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_TabComIfDescription">
        <attribute name="title">
         <string>COM Interface Description</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>21</number>
         </property>
         <property name="topMargin">
          <number>30</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="pc_VerticalLayout_3">
           <item>
            <widget class="stw_opensyde_gui::C_SdBueComIfDescriptionWidget" name="pc_WidgetComIfDescr" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeTawPageNavi</class>
   <extends>QTabWidget</extends>
   <header>C_OgeTawPageNavi.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdBueBusEditPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>C_SdBueBusEditPropertiesWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui::C_SdBueComIfDescriptionWidget</class>
   <extends>QWidget</extends>
   <header>C_SdBueComIfDescriptionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
